test/fixtures/templater/jinja_exclude_macro_path/macros/macros_exclude/macro_exclude.sql

```diff
@@ -0,0 +1,3 @@
+{% materialization my_materialization_name, default %}
+ -- materialization...
+{% endmaterialization %}
```

Comment:
I understand why you're adding this file, to check that the jinja templater doesn't error with the custom tag `materialization`. Can you add a comment in this file to document that?